docs/development/pull-requests.md

```diff
@@ -10,16 +10,19 @@ When contributing changes to this project, you must agree to the [DCO](DCO).
 Commits must include a `Signed-off-by:` header which certifies agreement with
 the terms of the [DCO](DCO).
 
-Using `-s` with `git commit` will automtaically add this header.
-
-## Merge Criteria
-
-Pull requests should receive at least one approval before merging. CI should also be passing. You may enable the [auto-merge feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request) on a PR to have it be automatically merged once this criteria has been met.
+Using `-s` with `git commit` will automatically add this header.
 
 ## Requesting Reviews
 
 Use the GitHub reviewers field to request reviews from a specific individual. Be sure to also select "re-request review" after making changes in response to a past review to ensure the reviewer knows it's ready for another look.
 
+## Merge Criteria
+
+* Pull requests should receive at least one approval before merging.
+* CI should also be passing.
+* Every person who is on the list of reviewers should be given a chance to weigh in before merging.
+* The `hold` label must not be present on the PR. Ensure that whoever added the `hold` is OK with removing the label before the PR is merged.
```

Comment:
Agree, the rebasing can get a bit heavy when review requests go on for too long.

Reply:
For people in the "reviewers" list, I think 24-hours is reasonable. Like everything, the importance is case-by-case depending on the PR. It may make sense to wait longer. It may be trivial enough that it doesn't matter. Use your best judgement.

For the `hold` label, I consider this a hard block with no time limit. You've got to work with whoever placed the label to get it removed.

Reply:
@russellb  SGTM!

Reply:
@russellb SGTM

Reply:
updated

Reply:
@russellb I think we should either setup the source code owner file or add a guideline that the PR author should atleast add one reviewer in the PR.